Makefile

```diff
@@ -5,7 +5,7 @@ PKG=github.com/guacsec/guac/pkg/version
 LDFLAGS="-X $(PKG).Version=$(VERSION) -X $(PKG).Commit=$(COMMIT) -X $(PKG).Date=$(BUILD)"
 
 CONTAINER ?= docker
-CPUTYPE=$(shell uname -m)
+CPUTYPE=$(shell uname -m | sed 's/x86_64/amd64/')
```

Comment:
this looks like a rebase issue. Should not be part of your PR.

Reply:
Corrected.

Reply:
Would that help?

Reply:
Ok. Should be fixed now.